apis/v1alpha1/opentelemetrycollector_types.go

```diff
@@ -248,6 +248,16 @@ type ScaleSubresourceStatus struct {
 	// OpenTelemetryCollector's deployment or statefulSet.
 	// +optional
 	Replicas int32 `json:"replicas,omitempty"`
+
+	// readyReplicas is the number of pods targeted by this OpenTelemetryCollector with a Ready Condition.
+	// +optional
+	ReadyReplicas int32 `json:"readyReplicas,omitempty"`
+
+	// statusReplicas is the number of pods targeted by this OpenTelemetryCollector
+	// with a Ready Condition and non-terminated pdos targeted by this OpenTelemetryCollector's
+	// deployment, statefulSet.
+	// +optional
+	StatusReplicas string `json:"statusReplicas,omitempty"`
```

Comment:
Why is it string and not a number?

Reply:
Due to printcolumns (kubebuilder) constraints, we cannot concatenate two int status values to present Ready -> 1/1, instead of creating two columns Current and Desired.